src/CollageImage.py

```diff
@@ -49,3 +55,12 @@ def Width(self):
     @Height.getter
     def Height(self):
         return self.photo_image.height()
+
+    def update_corners(self):
+        size = self.PIL_image.size
+        img = self.original.resize(size)
+        alpha = self.corners.get_alpha(size)
+        img.putalpha(alpha)
+        self.PIL_image = img
+        self.collage.itemconfig(self.Id, image=self.PhotoImage)
+        self.reset_selection()
```

Comment:
I think there is no need to reset selection, because photoImage size do not change